libnavigation-base/src/main/java/com/mapbox/navigation/base/routerefresh/MapboxRouteOptionsProvider.kt

```diff
@@ -1,21 +1,36 @@
-package com.mapbox.navigation.core.directions.session
+package com.mapbox.navigation.base.routerefresh
```

Comment:
+1

Reply:
it was done because I wouldn't expose Default implementation, there's was a provider of options providers interface. Now it's no sense, will move to `core` back